src/engine/state.ts

```diff
@@ -236,7 +238,9 @@ repository.on("update", ({added}: {added: TrustedEvent[]}) => {
     }
 
     if (event.kind === WRAP) {
-      unwrapper.push(event)
+      if (get(canDecrypt)) {
```

Comment:
Use `withGetter` when defining the store and `.get()` since this is a fairly hot code path.